miqa/learning/nn_classifier.py

```diff
@@ -284,7 +284,7 @@ def evaluate1(model, image_path):
         transform=train_transforms,
     )
     evaluation_loader = DataLoader(
-        evaluation_ds, batch_size=1, num_workers=1, pin_memory=torch.cuda.is_available()
+        evaluation_ds, batch_size=1, num_workers=0, pin_memory=torch.cuda.is_available()
```

Comment:
I think I'd prefer if we just omit the kwarg altogether; seems less confusing in terms of intent.